packages/app/src/app/overmind/internalActions.ts

```diff
@@ -615,6 +615,7 @@ export const initializeActiveWorkspace = async ({
       state.dashboard.teams = teams.me.workspaces;
       state.primaryWorkspaceId = teams.me.primaryWorkspaceId;
       state.userCanStartTrial = teams.me.eligibleForTrial;
+      state.userFeatureFlags = teams.me.featureFlags;
```

Comment:
I thought we should get it from workspace, is this expected?

Reply:
this is a separate flag at the user level, if the users are allowed to create new ubb workspaces